.changeset/empty-fans-reflect.md

```diff
@@ -0,0 +1,70 @@
+---
+'astro': minor
+---
+
+Adds a new `experimental.tsconfig` configuration option
```

Comment:
Note from me that Florian will now grab content from the finalized API documentation to refine the changeset! :raised_hands: 

I'm out until I'm summoned again to look at this!